base/docs/basedocs.jl

```diff
@@ -697,6 +697,13 @@ DevNull
 # doc strings for code in boot.jl and built-ins
 
 """
+    Void
+
+An ordinary type with no fields that is the type of [`nothing`](@ref).
```

Comment:
Not sure :smile:: I copied it from the [manual](https://docs.julialang.org/en/stable/manual/faq/#Nothingness-and-missing-values-1).